docs/architecture/coding-standards.md

```
@@ -0,0 +1,369 @@
+    action1: () => { /* implementation */ },
+    action2: () => { /* implementation */ },
+  };
+  
+  return { state, error, ...actions };
+}
+```
+
+## File Organization
+
+### Directory Structure
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)